experimenter/experimenter/klaatu/tests/test_client.py

```diff
@@ -0,0 +1,180 @@
+import datetime
+import unittest
+from io import BytesIO
+from unittest import mock
+
+from experimenter.klaatu.client import KlaatuClient
+
+
+class TestKlaatuClient(unittest.TestCase):
+    def setUp(self):
+        self.client = KlaatuClient(workflow_name="windows_manual.yml")
+
+    @mock.patch("requests.post")
```

Comment:
Usually you want to specify the full path at the CALL site, ie inside of 

```py
experimenter.klaatu.client
```

is where we want to mock, so here we would say

```py
@mock.patch('experimenter.klaatu.client.requests')
```